src/pages/submit/update.tsx

```diff
@@ -202,46 +228,54 @@ export default function Update() {
     if (state.validated) {
       submitUpdate()
     }
-  }, [state.validated])
+  }, [state.validated, isEdit, isEditAccepted])
 
   if (accountState.loading || updateState.loading) {
     return <LoadingView />
   }
 
-  if (updateId && (updateState.error || update?.status === UpdateStatus.Late || update?.status === UpdateStatus.Done)) {
+  const isDisabled =
+    !isEdit &&
+    updateId &&
+    (updateState.error || update?.status === UpdateStatus.Late || update?.status === UpdateStatus.Done)
+
+  const isUserEnabledToEdit = isEdit && update?.author === account
```

Comment:
Maybe we should let update authors and proposal coauthors edit.

Reply:
According to Gino, only the author of the update can edit it